diff --git a/node_modules/docusaurus-plugin-openapi/lib/openapi/createExample.js b/node_modules/docusaurus-plugin-openapi/lib/openapi/createExample.js
index 793113b..a866cbe 100644
--- a/node_modules/docusaurus-plugin-openapi/lib/openapi/createExample.js
+++ b/node_modules/docusaurus-plugin-openapi/lib/openapi/createExample.js
@@ -29,6 +29,7 @@ const primitives = {
     },
     object: {},
     array: {},
+    "null": {},
 };
 const sampleFromSchema = (schema = {}) => {
     let { type, example, allOf, properties, items } = schema;
